extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -58,6 +62,7 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     const rawJson = this.element.getAttribute('json');
     const jsonParameters = rawJson ? JSON.parse(rawJson) : {};
     const tfcd = jsonParameters['tfcd'];
+    const adTestOn = isInExperiment(this.element, MANUAL_EXPERIMENT_ID);
```

Comment:
I might be missing something, but I don't see `MANUAL_EXPERIMENT_ID` used other than `isInExperiment()`. Since both `MANUAL_EXPERIMENT_ID` and `isInExperiment()` are imported from `traffic-experiments`, so I think `MANUAL_EXPERIMENT_ID` can be private?